isb_web/main.py

```diff
@@ -57,6 +59,16 @@
 )
 
 
+@app.on_event("startup")
+def on_startup():
+    dao.connect_sqlmodel(isb_web.config.Settings().database_url)
+
+
+def get_session():
+    with dao.get_session() as session:
+        yield session
```

Comment:
This seemingly innocent bit is what lets us swap in the test db so we can write fully functioning FastAPI tests that hit a real server.